connector/Dockerfile

```
@@ -40,4 +41,9 @@ COPY ./connector/.env /app/.env
 
 # Replaces ENV Var statements so they don't overwrite existing ENV Vars
 RUN sed -ri 's/^\s*(\S+)=(.*)$/\1=${\1:-"\2"}/' .env
-ENTRYPOINT bash -c 'set -a && source /app/.env && set +a && exec java -Djava.util.logging.config.file=/app/logging.properties $JVM_ARGS -jar app.jar'
+
+# Change to non-root privilege
+RUN addgroup -S nonroot && adduser -S edc -G nonroot
+USER edc
```

```suggestion
USER edc:edc
```

If not explicitly provided, the Dockerfile `USER` command sets the group to `nogroup` (GID 65533).